JT/12_data_structures/linear_types_exercises/linear_types.adoc

```diff
@@ -0,0 +1,238 @@
+:icons: font
+== Array
+
+image::../resources/array.png[align="center",width=700]
+
+An Array is a data structure used to collect multiple data elements of the same data type into one variable. +
+
+*Arrays can be classified into different types:*
+
+* *One-Dimensional Array:* An Array with only one row of data elements is known as a One-Dimensional Array.
+It is stored in ascending storage location.
+* *Two-Dimensional Array:* An Array consisting of multiple rows and columns of data elements is called a Two-Dimensional Array.
```

Comment:
I would suggest to separate explanations for `Two-Dimensional Array` and `Multidimensional Array` and explanations or examples for each of them if you mentioned in that section both of them.